WebApiStart/Database/BookDb.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using WebApiStart.Model;
+
+namespace WebApiStart.Database
+{
+    public class BookDb
+    {
+        private static List<Book> _bookList = new List<Book>();
+        
+        
+        public List<Book> GetBooks()
+        {
+            using (StreamReader r = new StreamReader("DbFiles/book.json"))
+            {
+                string json = r.ReadToEnd();
+                _bookList = JsonConvert.DeserializeObject<List<Book>>(json);
+            }
+            return _bookList;
+        }
+
+        public bool AddBook(Book book)
+        {
+            //id++;
+            //book.Id = id;
+            int idCount = _bookList.Count();
+            book.Id = ++idCount;
```

Comment:
In this case....if you contain 5 books and if u delete the 2nd book and then you add another book then id will be again 5 so Count cannot be used